### ISSUE TITLE
Improve mobile experience

### ISSUE THREAD

**Issue by Gaya** [ThymeApp/thyme#208]
There are a couple of things not really working on mobile.

- [x] Settings sections do not open / collapse
- [x] Inputs zoom when they get focus

Go over the overall experience and improve where possible.

Hint for zoom issue:

```
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, maximum-scale=1, user-scalable=no">
```

**Comment by Gaya** [comment]
Someone said safari will not zoom if font size is bigger than `16px`

**Comment by Gaya** [comment]
Settings page might have been fixed in #210
